tests/xlsx/xlsx_export.test.ts

```diff
@@ -855,11 +855,30 @@ describe("Test XLSX export", () => {
         {
           dataSets: [{ dataRange: "Sheet1!B2:B" }, { dataRange: "Sheet1!C4:4" }],
           labelRange: "Sheet1!A2:A",
-          type: "pie",
+          type: "bar",
+          horizontal: true,
         },
         "3"
       );
-
+      createChart(
```

Comment:
I don't think it's the right place to test your export, the describe state `references with headers should be converted to references with fixed coordinates`. You don't really care about that, you just want a random snapshot with a simple doughtnut/horizontal chart right ? 

There's gotta be tests where we test that the charts are correctly exported. Tests where the charts aren't on an unbounded range, creating a freaking ***two thousands lines*** of snapshot change 😅